sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -365,9 +365,6 @@ private[hive] class HiveClientImpl(
         },
         schema = schema,
         partitionColumnNames = partCols.map(_.name),
-        sortColumnNames = Seq(), // TODO: populate this
-        bucketColumnNames = h.getBucketCols.asScala,
-        numBuckets = h.getNumBuckets,
```

Comment:
Can we explicitly set `bucketSpec` to `None` at here? Also, let's add a comment to explain the reason that we do not populate this variable at here? 

btw, I think we will not populate this info for a bucketed hive table because we are using different hash functions.